src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -368,6 +383,40 @@ export class CourseUpdateComponent implements OnInit {
         }
         return semesters;
     }
+
+    /**
+     * Opens the organizations modal used to select an organization to add
+     */
+    openOrganizationsModal() {
+        const modalRef = this.modalService.open(OrganizationSelectorComponent, { size: 'xl', backdrop: 'static' });
+        modalRef.componentInstance.organizations = this.courseOrganizations;
+        modalRef.closed.subscribe((organization) => {
+            if (organization !== undefined) {
+                if (this.courseOrganizations === undefined) {
+                    this.courseOrganizations = [];
+                }
+                this.courseOrganizations.push(organization);
+            }
+        });
+    }
+
+    /**
+     * Removes an organization from the course
+     * @param organization to remove
+     */
+    removeOrganizationFromCourse(organization: Organization) {
+        this.courseOrganizations = this.courseOrganizations.filter((o) => o.id !== organization.id);
+        /*
+        if (this.course.id !== undefined) {
+            this.organizationService.removeCourseFromOrganization(organization.id!, this.course.id).subscribe(
+                () => {
+                    this.course.organizations = this.course.organizations!.filter((o) => o.id !== organization.id);
+                },
+                (error: HttpErrorResponse) => this.jhiAlertService.error(error.message),
+            );
+        }
+         */
```

Comment:
I think you can remove the commented code :)

Reply:
Removed with https://github.com/ls1intum/Artemis/pull/2748/commits/ab2264066656190b61efae207cf9a1382a0104d4

Thanks for noticing :smiley: